### PR TITLE
chore: add Ij 2022.3 (#231)

### LINKED ISSUE #231: Add IJ 2022.3 verification
IJ 2022.3 is out

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #231

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'idea'
     id 'java'
-    id 'org.jetbrains.intellij' version '1.9.0'
+    id 'org.jetbrains.intellij' version '1.10.0'
```
Comment (jeffmaury) [inline build.gradle:4]: Should be reverted as it breaks the MacOS build

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (lstocchi) [reply build.gradle:4]: I'm not going to merge this as it doesn't work with 1.9 for me. I'm trying to figure out why 1.10 is failing. Same for tekton

File: build.gradle [hunk 2]
```diff
@@ -33,29 +33,43 @@ runIdeForUiTests {
...
-    compile {
+    runtimeClasspath {
         exclude group: 'org.slf4j', module: 'slf4j-api'
     }
     integrationTestImplementation.extendsFrom testImplementation
     integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
 }
 
 sourceSets {
+    main {
```
Comment (jeffmaury) [inline build.gradle:61]: Why do you need this ? It is the default by convention and other plugins don't require this

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**